api/README.md

```diff
@@ -26,36 +26,38 @@ curl 0.0.0.0:3000/api/health
 
 And you should see a response of 👍.
 
-Examples of valid API calls can be found by looking at the 
-routes in the `api.ts` script, or at the API calls made from 
+Examples of valid API calls can be found by looking at the
+routes in the `api.ts` script, or at the API calls made from
 the `ui` code in the `ScholarReader.tsx` file.
 
-## Configuring the server to access the database
-
-Several of the API endpoints assume the server has access to 
-a private database. The server code already includes _some_ 
-login information. However, you must get the password from 
-one of the database administrators (at the time of writing 
-this, andrewhead@berkeley.edu). Once you have this password, 
-create a file named `config/secret.json` with these 
-contents:
-
-```json
-{
-  "database": {
-    "password": "<password>"
-  }
-}
-```
+## One Time Setup
+
+The API uses several secrets at runtime to:
+
+- Access a database, where information about the papers and
+  the entities we extract from them is stored.
+- Query [Semantic Scholar's Public API](https://api.semanticscholar.org/).
+
+The API queries work without an API key, but will get rate
+limited if you make more than 100 queries in 5 minutes.
+This can happen relatively quickly during normal use of the
+application.
+
+The database won't workout without the appropriate
+configuration.
```

Comment:
Nit: What is the appropriate configuration? An example `secrets.json` file would be a useful reference so folks know what keys to use when plugging in the values without having to look at the TS interface.

Reply:
Sure, can do.

Reply:
Done, TAL.